.github/workflows/ci.yml

```diff
@@ -58,8 +63,13 @@ jobs:
             cargo pgrx init
           fi
 
+          git checkout master
+          echo "\q" | cargo pgrx run
+          psql -p 28816 -h localhost -d pgml -P pager -c "CREATE EXTENSION pgml;"
+          git checkout $CI_BRANCH
+          echo "\q" | cargo pgrx run
+          psql -p 28816 -h localhost -d pgml -P pager -c "ALTER EXTENSION pgml UPDATE;"
```

Comment:
intention for this is to just validate that `alter extension update` actually runs